src/test/java/eu/interop/federationgateway/service/CallbackTaskExecutorServiceTest.java

```diff
@@ -111,7 +111,7 @@ public void setup() throws IOException, NoSuchAlgorithmException, KeyManagementE
     );
 
     mockWebServer.start();
-    mockWebServer.useHttps(sslContext.getSocketFactory(), false);
+    mockWebServer.useHttps(sslContext.getSocketFactory());
```

Comment:
Is this intended looks like false becomes true?

Reply:
Was just a flag for a feature that never exist in the lib 😅
see https://github.com/square/okhttp/pull/7317